docs/metrics/logs-to-metrics.md

```diff
@@ -84,7 +84,9 @@ You can't use a scheduled view or an index in the scope of a Logs-to-Metrics rul
 
 ### Using Logs-to-Metrics in the frequent or infrequent tier
 
-If you want to create a Logs-to-Metrics rule for logs in the frequent or infrequent tier, you must create the rule with the same log search scope as the partition where the data lives, because you cannot use `_index`  in a Logs-to-Metrics rule. For example, if you have a partition, `_index=foo`, whose routing expression is `_sourceCategory=foo`, then you should use that same routing expression, `_sourceCategory=foo`, to scope the Logs-to-Metrics rule.
+Since `_index` is not accepted in logs-to-metrics rules you can just skip it and still get results for the data assigned to frequent and infrequent tier.
```

Comment:
```suggestion
Since `_index` is not accepted in logs-to-metrics rules, you can just skip it and still get results for the data assigned to frequent and infrequent tier.
```